index.html

```diff
@@ -106,47 +110,108 @@ <h1>About me</h1>
             Hello I’m a software developer! I can help you build a product ,
             feature or website Look through some of my work and experience! If
             you like what you see and have a project you need coded, don’t
-            hestiate to contact me.
+            hesitate to contact me.
           </p>
-          <div class="botton-about">
-             <button type="submit">Get My Resume</button>
-            </div>
-        
+          <div class="button-about">
+            <button type="submit">Get My Resume</button>
+          </div>
         </div>
-        <div class="lenguages-sp"> 
+        <div class="languages-sp">
           <div class="about-me">
-        <h4 class="titles2"><img src="images2/rectangle.png" alt="rectangulo1" />Language</h4>
+            <h4 class="titles2">
+              <img src="images2/rectangle.png" alt="rectangulo1" />Language
+            </h4>
             <ul>
-             <li>Javascript</li>
+              <li>Javascript</li>
               <li>Ruby</li>
               <li>Html</li>
               <li>Css</li>
             </ul>
           </div>
           <div class="about-me">
-          <h4 class="titles2"><img src="images2/rectangle2.png" alt="rectangulo1"/>Framworks</h4>
+            <h4 class="titles2">
+              <img src="images2/rectangle2.png" class="" alt="rectangulo1" />Framworks
+            </h4>
             <ul>
               <li>Bootstrap</li>
               <li>RubyonRails</li>
               <li>RSPec</li>
               <li>CopyBara</li>
               <li>Selenium</li>
             </ul>
-        </div>
+          </div>
 
           <div class="about-me">
-            <h4 class="titles2"><img src="images2/rectangle3.png" alt="rectangulo1"/>Skills</h4>
+            <h4 class="titles2">
+              <img src="images2/rectangle3.png" alt="rectangulo1" />Skills
+            </h4>
             <ul>
               <li>Codekit</li>
               <li>Github</li>
               <li>Codepen</li>
               <li>Gitlab</li>
               <li>Terminal</li>
+            </ul>
+          </div>
+        </div>
+      </section>
+      <section class="form">
+        <div>
+          <h6>
+            I'm always interested in hearing about new projects, so if you'd
+            like to chat please get in touch.
+          </h6>
```

Comment:
- [ ] You `h6` text is not matched with the Figma design. In the Figma design, there is more white space on the both left and right sides. Please work on it.

Your design             |  Figma Design
:-------------------------:|:-------------------------:
![2](https://user-images.githubusercontent.com/70483835/165421886-d540982e-552b-4540-8cb3-28e8c0c3564e.png) | ![1](https://user-images.githubusercontent.com/70483835/165421898-b82243c2-38ec-4e1d-ad60-c99527e0ecb4.png)